### DISCUSSION TITLE
Possible to Generate A Feels Like Temperature with Solar Radiation?

### DISCUSSION THREAD

**Discussion by atilt2** [Pirate-Weather/pirateweather#375] [Ideas]
I see the formula used for apparentTemperature doesn't include solar radiation, is there a way to provide this "Real Feel" temperature to include radiation as per [this comment](https://github.com/breezy-weather/breezy-weather/discussions/1085#discussioncomment-9734935)?

**Comment by cloneofghosts** [comment]
This should be possible to do. Will ping @alexander0042 to see if its something he can look at

**Comment by cloneofghosts** [comment]
I created issue #379 from this discussion for you.

**Comment by alexander0042** [comment]
Hi @atilt2- thanks for opening this discussion, and sorry for my slow reply here! I really think this is a great idea, and actually almost got it working (check out the unused function in the [code](https://github.com/Pirate-Weather/pirate-weather-code/blob/53b2e6c85dfd9dab074dacb5e475632f02c59997/API/responseLocal.py#L332)). 

The key issue here is pretty simple- what should solar radiation be? The [original paper](https://journals.ametsoc.org/view/journals/apme/23/12/1520-0450_1984_023_1674_ausoat_2_0_co_2.xml) is frustratingly vague on this factor, and while [this thread](https://github.com/open-meteo/open-meteo/discussions/651) suggests that 0.1 * Shortwave radiation (which we have), I don't like that they have to cap it at 550, since it suggests that something is wrong. 

Long story short- if someone can parse through those equations and work out how to calculate Q, I'd love to switch to that. From my own perspective, it definitely "feels" much warmer in the winter when the sun is out, so I think this would add to the accuracy of that equation.
